build-rpi-raspbian/first-boot.sh

```diff
@@ -83,6 +85,7 @@ apt update &>/dev/null
 apt --yes upgrade &>/dev/null
 if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
+sh /boot/webif.sh restart
```

Comment:
Why?

Reply:
in case the python process gets killed by a python package upgrade